test/test_sdist_package.py

```diff
@@ -0,0 +1,29 @@
+import shutil
+from pathlib import Path
+
+from . import utils
+
+
+def test(capfd, tmp_path):
+    sdist_filename = "spam-0.1.0.tar.gz"
+    sdist = Path(__file__).parent / sdist_filename
```

Comment:
Rather than including a tar.gz in the repo, I'd rather build this within the test, so we can all see what's actually being tested by reading the code.